### PR TITLE
Fix dailymotion specs

### PR DESCRIPTION
**Pull request by joaocv3**
- Update Dailymotion VCR cassetes
- Update Dailymotion specs
- Ignore Dailymotion thumbnail CDN subdomain in specs
- Change VCR re_record_interval to 28 days

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -12,7 +12,6 @@
   config.cassette_library_dir = 'spec/fixtures/vcr_cassettes'
   config.default_cassette_options = {
     record: :new_episodes,
-    re_record_interval: 7 * 24 * 60 * 60
```
Comment (thibaudgg) [inline spec/spec_helper.rb:15]: Do you think we shouldn't re-record these cassettes from time to time?

Comment (joaocv3) [reply spec/spec_helper.rb:15]: I spent some time thinking about it yesterday.

There are some nondeterministic values from APIs like Dailymotion. 

Dailymotion changes its CDNs subdomain every time:
```sh
 curl -s "https://api.dailymotion.com/video/x7lni3?fields=thumbnail_60_url" | jq '.thumbnail_60_url'                                    
"https://s1.dmcdn.net/v/mj4x1VKDs8zz5egS/x60"

curl -s "https://api.dailymotion.com/video/x7lni3?fields=thumbnail_60_url" | jq '.thumbnail_60_url'  
"https://s2.dmcdn.net/v/mj4x1VKDs8zz5egS/x60"
```

For this specific Dailymotion scenario, we could do some regex that will ignore the subdomain. We could also find the other scenarios where these details can be ignored.

Maybe we could have a longer interval, maybe couple weeks. 

By ignoring those details in specs and changing the re_record_interval to 3 or 4 weeks seems like a happy medium to me. What do you think @thibaudgg ?

Comment (thibaudgg) [reply spec/spec_helper.rb:15]: Yes 3/4 weeks sounds like a good idea, I think re-recording the cassettes from time to time and making sure with custom CDN regex they don't break seems like a good practice. That way we know when something is changing from the provider side and we can fix our code accordingly. What do you think @joaocv3?

Comment (joaocv3) [reply spec/spec_helper.rb:15]: I think it works great @thibaudgg 

I changed and tested it re-recording the Dailymotion cassetes a couple times, everything works fine with the regex strategy.

**Comment by thibaudgg** [comment]
Thanks @joaocv3!

**Pull request merged**